pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -3001,6 +3000,7 @@
     <EmbeddedResource Include="Model\Prosit\Config\PrositConfig.xml">
       <SubType>Designer</SubType>
     </EmbeddedResource>
+    <None Include="ClassDiagram1.cd" />

```

Comment:
This file contains changes we don't want. Please limit the changes to just those required for this fix.